database/db.go

```diff
@@ -13,6 +13,8 @@ type connection struct {
 	collection *mongo.Collection
 }
 
+var Conn *connection
+
 func NewConnection() *connection {
 	config := utils.GetConfig()
 	client, err := mongo.NewClient(config.MongoUrl)
```

Comment:
We should close the connection using `defer` at some point. When the backend dies, the DB connection should close - not exactly sure how that works at the moment.

Reply:
Yeah good call, I think we also need some way to establish connection again if it dies at any point.

Reply:
That would be interesting, if the connection doesn't exists, make a new connection. Yep.